### DISCUSSION TITLE
Localized description

### DISCUSSION THREAD

**Discussion by Shiaulis** [getsentry/sentry-cocoa#1883] [Q&A] [answered]
Hey folks! We are looking for a way to integrate your framework into our iOS app. One thing we cannot figure out still, is there any way to read our errors description? For now our error system is pretty complicated, we are using localized descriptions, as well as underlying errors. Is there any way to see those details in web? For now the only thing we see about error is its name and error code:
<img width="615" alt="image" src="https://user-images.githubusercontent.com/13601748/173640457-1082409f-398a-4343-bb36-fdb8bf1fc87b.png">

Did we miss some documentation or functionality that can help us?

**Comment by philipphofmann** [comment, marked as answer]
If you capture an `NSError` the error description in Sentry should be the error name and code.

<img width="547" alt="Screen Shot 2022-06-15 at 07 33 22" src="https://user-images.githubusercontent.com/2443292/173747116-5cef061d-3ddb-4ed3-af77-28f79bff4dd9.png">

We don't use localized descriptions in error titles because this would create different error groups and the descriptions are especially useless if you don't speak the language. Still, they should end up in the user info section below the stacktrace in Sentry.

Where in Sentry did you take your screenshot?
Which Cocoa SDK version are you using?
Can you paste a code snipped of a sample error and how you capture this error, please?

**Reply by Shiaulis** [reply]
[redacted]
Spending one more day on research of our issues helped to understand core problems with our current solution. We've figured out what was wrong on our side, now we can see all the information and it looks good on our side, including description and underlying errors

**Reply by philipphofmann** [reply]
Thanks for the update, @Shiaulis.
